airflow/providers/google/cloud/transfers/gcs_to_gcs.py

```diff
@@ -90,6 +90,9 @@ class GCSToGCSOperator(BaseOperator):
         If set as a sequence, the identities from the list must grant
         Service Account Token Creator IAM role to the directly preceding identity, with first
         account from the list granting this role to the originating account (templated).
+    :param source_object_required: When source_object_required is True, if you want to copy / move a specific blob
+        and it doesn't exist, an exception is raised and the task is marked as failed.
+        This parameter doesn't have any effect when the source_object that you pass is a folder or pattern.
```

Comment:
can we simplify it? `When ___ is true, if ....` is a sentence that is hard to follow.
You need to fix the sentence any way as it fails the static checks (line to long) so please try to simplify it.

Reply:
I've simplified the description. Thanks for the feedback!